### PR TITLE
Uptade README to add example for using public methods

### PR DESCRIPTION
**Pull request by yngrdyn**
Here's a small change to the README, with information on how to use a public method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,12 @@ export default {
 | render | –                                                                            | manually (re)render document         |
 | print  | print resolution (`number`), filename (`string`), all pages flag (`boolean`) | print document via browser interface |
 
+In order to use the public method you need to use ref.
+
+```vue
+  <vue-pdf-embed ref="myPdf" :source="pdfSource" />
+```
+
```
Comment (hrynko) [inline README.md:112]: I would keep it short and replace the code example with the following note:

```suggestion
**Note:** Public methods can be accessed via a [template ref](https://vuejs.org/guide/essentials/template-refs.html).

```

File: README.md [hunk 2]
```diff
@@ -116,6 +122,8 @@ Besides the component itself, the module also includes a `getDocument` function
 
 [Advanced Usage Demo (StackBlitz)](https://stackblitz.com/fork/vue-pdf-embed)
 
+[Public methods Usage Demo (JSFiddle)](https://jsfiddle.net/yngrdyn/eq73xyar/24/)
+
```
Comment (hrynko) [inline README.md:126]: Please remove this lines as this fiddle is not maintainable by the repo maintainers. Though I can reuse this and add to the Advanced Usage Demo afterwards.

**Review by hrynko — CHANGES_REQUESTED** [review]
Hi @yngrdyn,

Thanks for opening this PR. Please consider the following suggestions.

File: README.md (hunk 2 above)
Comment (yngrdyn) [reply README.md]: Please add this to `Advanced Usage Demo` fiddle. I think it adds value

**Review by hrynko — APPROVED** [review]
Thanks!

**Pull request merged**